components/file_preview_modal/file_preview_modal_image_controls/file_preview_modal_image_controls.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ChangeEvent, memo, useEffect, useRef, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {clamp} from 'lodash';
+
+import {PlusIcon, MinusIcon} from '@mattermost/compass-icons/components';
```

Comment:
@matthewbirtch the compass icon components are already being used by @shamboozles

Reply:
Ok, interesting @michelengelen. Do they always output as SVGs then? I thought they outputted as `i` tags with classes.

This is what the button tag HTML is showing for me:

<img width="368" alt="image" src="https://user-images.githubusercontent.com/2040554/186685444-85c02637-1d91-4ebe-8900-497ecdfd0b7d.png">

Reply:
@matthewbirtch yes, it is always outputting `svg` elements. This is due to a better ability to position the icon, since it is not controlled by font properties in css, but a block positioned element instead. 

Using `i` as an element to display icons is a remnant of old times and the usage is semantically incorrect, since the intention of the `i` tag is a different one. For reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i